v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FExtractions.scala

```diff
@@ -0,0 +1,256 @@
+    _.transform(s"enforing types to $tableName json values") { collection =>
+      collection.map {
+        case (jsonObj, filePath) =>
+          fieldNames.foldLeft((jsonObj, filePath)) {
+            case ((currentJsonObj, currentFilePath), fieldName) =>
+              val jsonValue = currentJsonObj
+                .apply(fieldName)
+                .fold(
+                  throw new Exception(
+                    s"convertJsonFieldsValueType: error when calling apply on $fieldName"
+                  )
+                ) { json =>
+                  convertJsonString(fieldName, json)
+                }
+              val transformedJson = addField(currentJsonObj, fieldName, jsonValue)
+              (transformedJson, currentFilePath)
+          }
+      }
+    }
+
+  /**
+    * converts a json string to a json double
+    *
+    * @param fieldName the field name of the json value being converted
+    */
+  def jsonStringToJsonDouble(fieldName: String, json: Json): Json =
+    json.asString
+      .fold(
+        throw new Exception(
+          s"jsonStringToJsonDouble: error when converting $fieldName value, $json, from type json string to type string"
+        )
+      ) {
+        case "." => Json.fromString("nan")
+        case ""  => Json.fromString("nan")
+        case str =>
+          Json.fromDoubleOrString(
+            Try(str.toDouble)
+              .getOrElse(
+                throw new Exception(
+                  s"jsonStringToJsonDouble: error when converting $fieldName value, $json, from type string to type double"
+                )
+              )
+          )
+      }
+
+  /**
+    * converts a json string to a json int
+    *
+    * @param fieldName the field name of the json value being converted
+    */
+  def jsonStringToJsonInt(fieldName: String, json: Json): Json =
+    json.asString
+      .fold(
+        throw new Exception(
+          s"jsonStringToJsonInt: error when converting $fieldName value, $json, from type json string to type string"
+        )
+      ) {
+        case "." => Json.fromString("nan")
+        case ""  => Json.fromString("nan")
+        case str =>
+          Json.fromInt(
+            Try(Integer.getInteger(str))
+              .getOrElse(
+                throw new Exception(
+                  s"jsonStringToJsonInt: error when converting $fieldName value, $json, from type string to type int"
+                )
+              )
+          )
+      }
+
+  /**
+    * converts a json string to a json boolean
+    *
+    * @param fieldName the field name of the json value being converted
+    */
+  def jsonStringToJsonBoolean(fieldName: String, json: Json): Json =
+    json.asString
+      .fold(
+        throw new Exception(
+          s"jsonStringToJsonBoolean: error when converting $fieldName value, $json, from type json string to type string"
+        )
+      ) { str =>
+        Json.fromBoolean(str == "1")
```

Comment:
Maybe do `str == "1" || str == "true"`? I think some of the boolean columns actually use true/false